homeassistant/components/homematicip_cloud/light.py

```diff
@@ -104,6 +120,65 @@ async def async_turn_off(self, **kwargs: Any) -> None:
         await self._device.turn_off_async()
 
 
+class HomematicipLightHS(HomematicipGenericEntity, LightEntity):
+    """Representation of the HomematicIP light with HS color mode."""
+
+    _attr_color_mode = ColorMode.HS
+    _attr_supported_color_modes = {ColorMode.HS}
+
+    def __init__(self, hap: HomematicipHAP, device, channel_index: int) -> None:
+        """Initialize the light entity."""
+        super().__init__(hap, device, channel=channel_index, is_multi_channel=True)
+
+    @property
+    def is_on(self) -> bool:
+        """Return true if light is on."""
+        return self.functional_channel.on
+
+    @property
+    def brightness(self) -> int | None:
+        """Return the current brightness."""
+        return (int)(self.functional_channel.dimLevel * 255.0)
```

Comment:
`(int)`? This looks like some casting, but I don't believe this results in casting

Reply:
uups my bad. fixed it.